src/main/java/net/minecraftforge/event/TickEvent.java

```diff
@@ -45,9 +48,24 @@ public TickEvent(Type type, LogicalSide side, Phase phase)
     }
 
     public static class ServerTickEvent extends TickEvent {
-        public ServerTickEvent(Phase phase)
+
+        private final BooleanSupplier haveTime;
+
+        public ServerTickEvent(Phase phase, BooleanSupplier haveTime)
         {
             super(Type.SERVER, LogicalSide.SERVER, phase);
+            this.haveTime = haveTime;
+        }
+
+        /**
+         * Returns whether the server has enough time to perform any
+         * additional tasks (usually IO related) during the current tick.
+         * 
+         * @return <code>true</code> if there is time, <code>false</code> otherwise
+         */
```

Comment:
We could simplify it a bit with the use of the inline `{@return}` tag:
```suggestion
        /**
         * {@return {@code true} whether the server has enough time to perform any additional
         * tasks (usually IO related) during the current tick, otherwise {@code false}}
         */
```

Reply:
It would seem Eclipse doesn't know what that is...

Reply:
Hmm, then Eclipse may not yet have updated itself with the new tag which was introduced in Java 16 ([JDK-8075778](https://bugs.openjdk.java.net/browse/JDK-8075778)). Remove the outermost braces to make it a block tag for now.